docs/sdk/serverless-signing/uploading.md

```diff
@@ -0,0 +1,230 @@
+:::
+
+<details>
+<summary>Click here for full code implementation</summary>
+<p>
+
+```ts
+import { LitNodeClient } from "@lit-protocol/lit-node-client";
+import { LIT_RPC, LitNetwork } from "@lit-protocol/constants";
+import {
+  createSiweMessageWithRecaps,
+  generateAuthSig,
+  LitAbility,
+  LitActionResource,
+} from "@lit-protocol/auth-helpers";
+
+const ethersSigner = new ethers.Wallet(
+    process.env.ETHEREUM_PRIVATE_KEY,
+    new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+);
+
+const litNodeClient = new LitNodeClient({
+    litNetwork: LitNetwork.DatilDev,
+    debug: false,
+});
+await litNodeClient.connect();
+
+const sessionSigs = await litNodeClient.getSessionSigs({
+    chain: "ethereum",
+    expiration: new Date(Date.now() + 1000 * 60 * 60 * 24).toISOString(), // 24 hours
+    resourceAbilityRequests: [
+        {
+            resource: new LitActionResource("*"),
+            ability: LitAbility.LitActionExecution,
+        },
+    ],
+    authNeededCallback: async ({
+        resourceAbilityRequests,
+        expiration,
+        uri,
+    }) => {
+        const toSign = await createSiweMessageWithRecaps({
+            uri: uri!,
+            expiration: expiration!,
+            resources: resourceAbilityRequests!,
+            walletAddress: ethersSigner.address,
+            nonce: await litNodeClient.getLatestBlockhash(),
+            litNodeClient,
+        });
+
+        return await generateAuthSig({
+            signer: ethersSigner,
+            toSign,
+        });
+    },
+});
+
+const litActionCode = `
+(async () => {
+  console.log("This is my Lit Action!");
+})();
+`;
+
+await litNodeClient.executeJs({
+    sessionSigs,
+    code: litActionCode,
+});
+```
+
+</p>
+</details>
+
+For this guide we'll focus on the very last couple of lines from the above full implementation:
+
+```ts
+const litActionCode = `
+(async () => {
+  console.log("This is my Lit Action!");
+})();
+`;
+
+await litNodeClient.executeJs({
+    sessionSigs,
+    code: litActionCode,
+});
+```
+
+First we're declaring our Lit Action as a [Immediately Invoked Function Expression](https://developer.mozilla.org/en-US/docs/Glossary/IIFE), i.e. a function that executes itself. This Lit Action is simple logging a string to the console, so you'd replace the `console.log` with your custom code.
+
+Next we're calling the `executeJs` method to create a request to the Lit network to execute our Lit Action. We provide our `litActionCode` string as the `code` parameter, and when the Lit network receives our request, our `string` will be parsed and executed on each Lit node.
+
+## Uploading to IPFS
```

Comment:
should we add a note that this is currently broken on mainnet?

Reply:
It was mentioned during the All-Team that a fix is already being tested, I think this is alright, especially with ETH Online people probably developing on the testnets. I'd be more worried about forgetting about the note 😅